### DISCUSSION TITLE
Accidentally pressed animejanai_v2_encode_single.bat, will it decrease my quality?

### DISCUSSION THREAD

**Discussion by PinkiePieFTW** [the-database/mpv-upscale-2x_animejanai#23] [Q&A]
A noob question, but I know this file is for lower GPUs (I have RTX3080) and I accidentally opened it. Will it decrease the performance?

**Comment by the-database** [comment]
That script is only used if you want to encode upscaled videos, so it's completely separate from anything that you play in mpv. 

To use that script you have to edit it to choose the video you want to encode, so if you didn't edit the bat file first, then running the script would have no effect at all.

**Reply by PinkiePieFTW** [reply]
Very much appreciated! Thanks.
